dask/array/slicing.py

```diff
@@ -1921,7 +1921,7 @@ def value_indices_from_1d_int_index(dim, vsize, loc0, loc1):
         # Note which dimension, if any, has 1-d integer array index
         dim_1d_int_index = None
 
-        for dim, (index, full_size, (loc0, loc1)) in enumerate(
+        for dim, (index, _, (loc0, loc1)) in enumerate(
```

Comment:
Similarly let's remove `array_shape` from the `zip` iteration here. I can't leave a direct suggestion in GitHub because the line below wasn't modified in this PR.